tests/end-to-end/_files/requires_environment_variable/SomeTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PHPUnit\TestFixture\requires_environment_variable;
+
+use PHPUnit\Framework\Attributes\RequiresEnvironmentVariable;
+use PHPUnit\Framework\TestCase;
+
+final class SomeTest extends TestCase
+{
+    #[RequiresEnvironmentVariable('FOO', 'bar')]
+    public function testOneShouldNotRun(): void
+    {
+        self::fail();
+    }
+
+    #[RequiresEnvironmentVariable('BAR')]
+    public function testTwoShouldNotRun(): void
+    {
+        self::fail();
+    }
+
+    #[RequiresEnvironmentVariable('FOO')]
+    public function testOneShouldRun(): void
+    {
+        self::assertTrue(true);
+    }
+
+    #[RequiresEnvironmentVariable('FOO', '1')]
+    public function testTwoShouldRun(): void
+    {
+        self::assertTrue(true);
+    }
+}
```

Comment:
> not sure what @mvorisek meant https://github.com/sebastianbergmann/phpunit/pull/6074#discussion_r1875896248 by "in separate process"?

https://docs.phpunit.de/en/10.5/annotations.html#runinseparateprocess

Reply:
I've added on test with `#[RunInSeparateProcess]`, is it good enough?